backend/src/main/java/de/jsmenues/backend/zabbixapi/ZabbixUser.java

```diff
@@ -29,6 +29,13 @@ public static ZabbixUser loadFromRepository(IConfigurationRepository configurati
         String password = configurationRepository.getVal(KEY_PASSWORD);
         String zabbixUrl = configurationRepository.getVal(KEY_URL);
 
+        // use standard values if none exist in redis
+        if (username == "" && password == "" && zabbixUrl == ""){
```

Comment:
this only works by accident.
strings needs to be compared with .equals().
== on strings compares the ids of the string objects, not the string contents.

here this works because the configurationRepository-framework returns "" when it does not find the value which results in "" == "". This is evaluated to true only because java uses the same object for both "" strings.